packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -553,47 +552,7 @@ export class ChatController {
             icon: 'list-add' as MynahIconsType,
         })
 
-        // const lspClientReady = await LspClient.instance.waitUntilReady()
```

Comment:
this is the old chat code path. We no longer start the indexing library for it. Hence we just remove this code to minimize our change in https://github.com/aws/aws-toolkit-vscode/pull/7480